polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -30,6 +31,8 @@ import {
 } from './components';
 import styles from './Frame.scss';
 
+const SCROLL_ELEMENT_HEIGHT = 56;
```

Comment:
How was this calculated? Can we have a comment here, or even better; use spacing values to do the calculation?

Also, if this is meant to match a CSS calculation, it would be good to link that in either code (preferable so it doesn't get out of date) or via a comment.

Reply:
We only use spacing tokens for padding, margin, and gap. Width and height should be hard-coded values. 56px is the topbar's fixed height.

Reply:
The name `SCROLL_ELEMENT_HEIGHT` is a bit generic to me since it's related to the ContextualSaveBar. 
Can we add `Contextual Save Bar` to the const name somewhere, please?